app/renderer/js/components/server-tab.ts

```diff
@@ -10,9 +10,8 @@ class ServerTab extends Tab {
 
 	template(): string {
 		return `<div class="tab" data-tab-id="${this.props.tabIndex}">
-					<div class="server-tooltip" style="display:none">${this.props.name}</div>
```

Comment:
Any reason to remove this?

Reply:
No, we can't! Unfortunately we can't layer any HTML/CSS over BrowserViews, so this will not be possible.